updated  adobe reader version and made words to bold (#9196)
MicrosoftDocs/windows-itpro-docs: windows/deployment/deploy-windows-mdt/deploy-a-windows-10-image-using-mdt.md

**Applies to**
-   Windows 10

This topic will show you how to take your reference image for Windows 10 (that was just [created](create-a-windows-10-reference-image.md)), and deploy that image to your environment using the **Microsoft Deployment Toolkit (MDT)**. 

> We will prepare for this by creating an **MDT** deployment share that is used solely for image deployment. Separating the processes of creating reference images from the processes used to deploy them in production allows greater control of on both processes. We will configure **Active Directory** permissions, configure the deployment share, create a new task sequence, and add applications, drivers, and rules.

Reviewer: ```suggestion
We will prepare for this by creating an **MDT** deployment share that is used solely for image deployment. Separating the processes of creating reference images from the processes used to deploy them in production allows greater control of both processes. We will configure **Active Directory** permissions, configure the deployment share, create a new task sequence, and add applications, drivers, and rules.
```

> For the purposes of this topic, we will use four computers: **DC01, MDT01, HV01 and PC0005**. 

Reviewer: ```suggestion
For the purposes of this topic, we will use four computers: **DC01**, **MDT01**, **HV01**, and **PC0005**. 
```

- **DC01** is a domain controller 
- **MDT01** is a domain member server 
- **HV01** is a Hyper-V server 
- **PC0005** is a blank device to which we will deploy Windows 10

> **MDT01** and **PC0005** are members of the domain contoso.com for the fictitious Contoso Corporation.  **HV01** used to test deployment of **PC0005** in a virtual environment.

Reviewer: ```suggestion
**MDT01** and **PC0005** are members of the domain contoso.com for the fictitious Contoso Corporation.  **HV01** is used to test the deployment of **PC0005** in a virtual environment.
```

The steps for creating the deployment share for production are the same as when you created the deployment share for creating the custom reference image:

1. Ensure you are signed on as: contoso\administrator.
2. In the **Deployment Workbench** console, right-click **Deployment Shares** and select **New Deployment Share**.
3. On the **Path** page, in the **Deployment share path** text box, type **D:\\MDTProduction** and click **Next**.
4. On the **Share** page, in the **Share name** text box, type **MDTProduction$** and click **Next**.
5. On the **Descriptive Name** page, in the **Deployment share description** text box, type **MDT Production** and click **Next**.
6. On the **Options** page, accept the default settings and click **Next** twice, and then click **Finish**.
7. Using **File Explorer**, verify that you can access the **\\\\MDT01\\MDTProduction$** share.

### Configure permissions for the production deployment share

> To read files in the deployment share, you need to assign **NTFS** and **SMB** permissions to the **MDT Build Account (MDT\_BA)** for the **D:\\MDTProduction** folder

Reviewer: ```suggestion
To read files in the deployment share, you need to assign **NTFS** and **SMB** permissions to the **MDT Build Account (MDT\_BA)** for the **D:\\MDTProduction** folder.
```

1.  Using the **Deployment Workbench**, expand the **Deployment Shares** node, and then expand **MDT Production**; select the **Operating Systems** node, and create a **folder** named **Windows 10**.
2.  Right-click the **Windows 10** folder and select **Import Operating System**.
3.  On the **OS Type** page, select **Custom image file** and click **Next**.
4.  On the **Image** page, in the **Source file** text box, browse to **D:\\MDTBuildLab\\Captures\\REFW10X64-001.wim** and click **Next**.
5.  On the **Setup** page, select the **Copy Windows 7, Windows Server 2008 R2, or later setup files from the specified path** option; in the **Setup source directory** text box, browse to **D:\\MDTBuildLab\\Operating Systems\\W10EX64RTM** and click **Next**.
6.  On the **Destination** page, in the **Destination directory name** text box, type **W10EX64RTM**, click **Next** twice, and then click **Finish**.
7.  After adding the **Operating System**, double-click the added **Operating System** name in the **Operating Systems / Windows 10** node and change the name to **Windows 10 Enterprise x64 RTM Custom Image**.

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> >The reason for adding the setup files has changed since earlier versions of **MDT**. **MDT 2010** used the setup files to install Windows. **MDT** uses **DISM** Command to apply the image; however, you still need the **setup files** because some components in **roles and features** are stored outside the main image.

Reviewer: ```suggestion
> The reason for adding the setup files has changed since earlier versions of **MDT**. **MDT 2010** used the setup files to install Windows. **MDT** uses the **DISM** command to apply the image; however, you still need the **setup files** because some components in **roles and features** are stored outside the main image.
```

 

![Imported OS](../images/fig2-importedos.png)

## Step 4: Add an application

> When you configure your **MDT Build Lab deployment** share, you can also add a**pplications** to the new deployment share before creating your task sequence. This section walks you through the process of adding an application to the **MDT Production deployment** share using **Adobe Reader** as an example.

Reviewer: ```suggestion
When you configure your **MDT Build Lab deployment** share, you can also add **applications** to the new deployment share before creating your task sequence. This section walks you through the process of adding an application to the **MDT Production deployment** share using **Adobe Reader** as an example.
```

### Create the install: Adobe Reader DC

On **MDT01**:

1. Download the Enterprise distribution version of [**Adobe Acrobat Reader DC**](https://get.adobe.com/reader/enterprise/) (AcroRdrDC2100120140_en_US.exe) to **D:\\setup\\adobe** on MDT01.

> 2. Extract the **.exe** file that you downloaded to an **.msi** (ex: .\AcroRdrDC2100120140_en_US.exe -sfx_o"d:\setup\adobe\install\" -sfx_ne).

Reviewer: ```suggestion
2. Extract the **.exe** file that you downloaded to a **.msi** (ex: .\AcroRdrDC2100120140_en_US.exe -sfx_o"d:\setup\adobe\install\" -sfx_ne).
```

## Step 5: Prepare the drivers repository

> In order to deploy Windows 10 with **MDT** successfully, you need drivers for the boot images and for the actual operating system. This section will show you how to add drivers for the boot image and operating system, using the following hardware models as examples:

Reviewer: ```suggestion
In order to deploy Windows 10 with **MDT** successfully, you need drivers for the boot images and for the actual operating system. This section will show you how to add drivers for the boot images and operating system, using the following hardware models as examples:
```

    -   Microsoft Corporation
        -   Surface Laptop

> The preceding folder names should match the actual make and model values that MDT reads from devices during deployment. You can find out the model values for your machines by using the following command in **Windows PowerShell Prompt:

Reviewer: ```suggestion
The preceding folder names should match the actual make and model values that MDT reads from devices during deployment. You can find out the model values for your machines by using the following command in an elevated **Windows PowerShell prompt**:
```

### Create the selection profiles for boot image drivers

By default, MDT adds any storage and network drivers that you import to the boot images. However, you should add only the drivers that are necessary to the boot image. You can control which drivers are added by using selection profiles.

> The drivers that are used for the boot images (Windows PE) are Windows 10 drivers. If you can’t locate **Windows 10** drivers for your device, a **Windows 7 or Windows 8.1** driver will most likely work, but Windows 10 drivers should be your first choice.

Reviewer: ```suggestion
The drivers that are used for the boot images (Windows PE) are Windows 10 drivers. If you can’t locate **Windows 10** drivers for your device, a **Windows 7** or **Windows 8.1** driver will most likely work, but Windows 10 drivers should be your first choice.
```

On **MDT01**:

1.  In the **Deployment Workbench**, under the **MDT Production** node, expand the **Advanced Configuration** node, right-click the **Selection Profiles** node, and select **New Selection Profile**.

> 2.  In the New **Selection Profile Wizard**, create a **selection profile** with the following settings:

Reviewer: ```suggestion
2.  In the **New Selection Profile Wizard**, create a **selection profile** with the following settings:
```

    1.  Selection Profile name: **WinPE x86**

>     2.  Folders: Select the **WinPE x86 folder** in **Out-of-Box** Drivers.

Reviewer: ```suggestion
    2.  Folders: Select the **WinPE x86 folder** in **Out-of-Box Drivers**.
```

>     3. Click **Next**, **Next** and **Finish**.

Reviewer: ```suggestion
    3. Click **Next**, **Next**, and **Finish**.
```

3.  Right-click the **Selection Profiles** node again, and select **New Selection Profile**.

> 4.  In the New **Selection Profile Wizard**, create a **selection profile** with the following settings:

Reviewer: ```suggestion
4.  In the **New Selection Profile Wizard**, create a **selection profile** with the following settings:
```

    1.  Selection Profile name: **WinPE x64**

>     2.  Folders: Select the **WinPE x64 folder** in **Out-of-Box** Drivers.

Reviewer: ```suggestion
    2.  Folders: Select the **WinPE x64 folder** in **Out-of-Box Drivers**.
```

>     3.  Click **Next**, **Next** and **Finish**.

Reviewer: ```suggestion
    3.  Click **Next**, **Next**, and **Finish**.
```

### Extract and import drivers for the x64 boot image

> **Windows PE** supports all the hardware models that we have, but here you learn to add boot image drivers to accommodate any new hardware that might require additional drivers. In this example, you add the latest Intel network drivers to the x64 boot image.

Reviewer: ```suggestion
**Windows PE** supports all the hardware models that we have, but here you learn how to add boot image drivers to accommodate any new hardware that might require additional drivers. In this example, you add the latest Intel network drivers to the x64 boot image.
```

On **MDT01**:

1. Download **PROWinx64.exe** from Intel.com (ex: [PROWinx64.exe](https://downloadcenter.intel.com/downloads/eula/25016/Intel-Network-Adapter-Driver-for-Windows-10?httpDown=https%3A%2F%2Fdownloadmirror.intel.com%2F25016%2Feng%2FPROWinx64.exe)).

> 2.  Extract **PROWinx64.exe** to a temporary folder - in this example to the **C:\\Tmp\\ProWinx64** folder.

Reviewer: ```suggestion
2.  Extract **PROWinx64.exe** to a temporary folder - in this example to the **C:\\Tmp\\ProWinx64** folder.
```
```suggestion
2.  Extract **PROWinx64.exe** to a temporary folder - in this example to the **C:\\Tmp\\ProWinx64** folder. Note that extracting the **.exe** file manually requires an extraction utility. You can also run the .exe and it will self-extract files to the **%userprofile%\AppData\Local\Temp\RarSFX0** directory. This directory is temporary and will be deleted when the **.exe** terminates.
```

>     a. **Note**: Extracting the **.exe** file manually requires an extraction utility. You can also run the .exe and it will self-extract files to the **%userprofile%\AppData\Local\Temp\RarSFX0** directory. This directory is temporary and will be deleted when the **.exe** terminates. 

Reviewer: ```suggestion
```

3.  Using **File Explorer**, create the **D:\\Drivers\\WinPE x64\\Intel PRO1000** folder.
4.  Copy the content of the **C:\\Tmp\\PROWinx64\\PRO1000\\Winx64\\NDIS64** folder to the **D:\\Drivers\\WinPE x64\\Intel PRO1000** folder.

> 5.  In the **Deployment Workbench**, expand the **MDT Production** > **Out-of-Box Drivers** node, right-click the **WinPE x64** node, and select **Import Drivers**, and use the following Driver source directory to import drivers: **D:\\Drivers\\WinPE x64\\Intel PRO1000**.

Reviewer: ```suggestion
5.  In the **Deployment Workbench**, expand the **MDT Production** > **Out-of-Box Drivers** node, right-click the **WinPE x64** node, select **Import Drivers**, and use the following driver source directory to import drivers: **D:\\Drivers\\WinPE x64\\Intel PRO1000**.
```
